run-tests-with-build-business-central-community/pom.xml

```diff
@@ -0,0 +1,156 @@
+                        <configuration>
+                            <connectionUrl>scm:git:${kie.wb.common.repo.url}</connectionUrl>
+                            <checkoutDirectory>${sources.directory}/kie-wb-common</checkoutDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>checkout-optaplanner-wb</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>checkout</goal>
+                        </goals>
+                        <configuration>
+                            <connectionUrl>scm:git:${optaplanner.wb.repo.url}</connectionUrl>
+                            <checkoutDirectory>${sources.directory}/optaplanner-wb</checkoutDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.kie</groupId>
+                <artifactId>unpack-build-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>download-build-jars</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>unpack-build</goal>
+                        </goals>
+                        <configuration>
+                            <excludeDirectories combine.children="append">
+                                <excludeDirectory>.*kie-wb-common-dmn.*</excludeDirectory>
+                                <excludeDirectory>.*kie-wb-common-forms.*</excludeDirectory>
+                                <excludeDirectory>.*kie-wb-common-stunner.*</excludeDirectory>
+                                <excludeDirectory>.*kie-wb-common-dynamic-forms.*</excludeDirectory>
+                                <excludeDirectory>.*optaplanner-wb-ui.*</excludeDirectory>
+                                <excludeDirectory>.*^kie-server.*</excludeDirectory>
+                                <excludeDirectory>.*^guvnor.*-client</excludeDirectory>
+                                <excludeDirectory>.*^jbpm-console-ng.*-client</excludeDirectory>
+                                <excludeDirectory>.*optaplanner-wb-ui.*</excludeDirectory>
+                                <excludeDirectory>.*drools-workbench-models-guided-template.*</excludeDirectory>
+                                <excludeDirectory>.*kie-wb-common-services-backend.*</excludeDirectory>
+                            </excludeDirectories>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-invoker-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>precompile</id>
```

Comment:
In original test suite there is no precompile phase used. Is this necessary? We can talk about this in person.